2018-07-28 14:35:58,109|INFO |o.a.http.impl.execchain.RetryExec|I/O exception (java.net.SocketException) caught when processing request to {}->http://121.40.164.61:8080: 网络不可达 (connect failed)
2018-07-28 14:35:58,109|INFO |o.a.http.impl.execchain.RetryExec|Retrying request to {}->http://121.40.164.61:8080
2018-07-28 14:35:59,155|INFO |o.a.http.impl.execchain.RetryExec|I/O exception (java.net.SocketException) caught when processing request to {}->http://121.40.164.61:8080: 网络不可达 (connect failed)
2018-07-28 14:35:59,155|INFO |o.a.http.impl.execchain.RetryExec|Retrying request to {}->http://121.40.164.61:8080
2018-07-28 14:36:00,179|INFO |o.a.http.impl.execchain.RetryExec|I/O exception (java.net.SocketException) caught when processing request to {}->http://121.40.164.61:8080: 网络不可达 (connect failed)
2018-07-28 14:36:00,179|INFO |o.a.http.impl.execchain.RetryExec|Retrying request to {}->http://121.40.164.61:8080
2018-07-28 14:36:03,251|INFO |o.a.http.impl.execchain.RetryExec|I/O exception (java.net.SocketException) caught when processing request to {}->http://121.40.164.61:8080: 网络不可达 (connect failed)
2018-07-28 14:36:03,251|INFO |o.a.http.impl.execchain.RetryExec|Retrying request to {}->http://121.40.164.61:8080
2018-07-28 14:36:06,323|ERROR|esign.utils.httpclient.f|http to url failed. url:http://121.40.164.61:8080/tgmonitor/rest/app!getAPIInfo2
2018-07-28 14:36:06,323|ERROR|esign.utils.httpclient.f|http client execute failed.
java.net.SocketException: 网络不可达 (connect failed)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:141)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107)
	at esign.utils.httpclient.f.a(HttpUtil.java:557)
	at esign.utils.httpclient.f.a(HttpUtil.java:526)
	at esign.utils.httpclient.f.a(HttpUtil.java:520)
	at esign.utils.httpclient.f.a(HttpUtil.java:230)
	at esign.utils.modeladapter.impl.rpc.a.a(ProjectUrlsRpc.java:34)
	at esign.utils.modeladapter.impl.f.a(RefreshModels.java:48)
	at esign.utils.modeladapter.impl.g.run(RefreshModelsTask.java:31)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
2018-07-28 14:36:06,334|ERROR|esign.utils.modeladapter.impl.g|refresh url into models failed.
esign.utils.exception.ai: 外部服务错误
	at esign.utils.exception.collector.meta.a.a(ErrorMeta.java:63)
	at esign.utils.httpclient.f.a(HttpUtil.java:581)
	at esign.utils.httpclient.f.a(HttpUtil.java:526)
	at esign.utils.httpclient.f.a(HttpUtil.java:520)
	at esign.utils.httpclient.f.a(HttpUtil.java:230)
	at esign.utils.modeladapter.impl.rpc.a.a(ProjectUrlsRpc.java:34)
	at esign.utils.modeladapter.impl.f.a(RefreshModels.java:48)
	at esign.utils.modeladapter.impl.g.run(RefreshModelsTask.java:31)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: java.net.SocketException: 网络不可达 (connect failed)
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.socket.PlainConnectionSocketFactory.connectSocket(PlainConnectionSocketFactory.java:74)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:141)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184)
	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88)
	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107)
	at esign.utils.httpclient.f.a(HttpUtil.java:557)
	... 8 common frames omitted
